specs/reducers/conditional_spec.js

```diff
@@ -0,0 +1,284 @@
+import Immutable from "immutable";
+import {
+  skipStatement,
+  sequenceStatement,
+  bindingStatement,
+  conditionalStatement,
+} from "../samples/statements";
+import {
+  lexicalVariable,
+  lexicalNumber,
+  lexicalRecord,
+} from "../samples/lexical";
+import {
+  buildState,
+  buildStack,
+  buildSemanticStatement,
+  buildStore,
+  buildEquivalenceClass,
+  buildVariable,
+  buildEnvironment,
+} from "../../app/oz/machine/build";
+import reduce from "../../app/oz/reducers/conditional";
+
+describe("Reducing X=VALUE statements", () => {
```

Comment:
This `describe` call here should say something like `"Reducing if statements"`

Reply:
changed